kms++/src/card.cpp

```diff
@@ -148,6 +148,21 @@ Card::Card(const std::string& driver, uint32_t idx)
 	setup();
 }
 
+Card::Card(int fd, bool take_ownership)
+{
+	if (take_ownership) {
+		m_fd = fd;
+	}
+	else {
+		m_fd = fcntl(fd, F_DUPFD_CLOEXEC, 0);
```

Comment:
Can you fix the formatting here to } else {

Reply:
Sure. Stand by for an update to the PR.